api/views/workflow.py

```diff
@@ -33,14 +36,13 @@ def get_queryset(self):
 
 
 class SourceViewSet(viewsets.ReadOnlyModelViewSet):
+    filter_backends = (filters.OrderingFilter, )
+    ordering = ('id', )
+    ordering_fields = ('title', )
     serializer_class = SourceSerializer
 
     def get_queryset(self):
-        queryset = Source.objects.exclude(icon='').exclude(is_deleted=True)
-        sort = self.request.query_params.get('sort')
```

Comment:
Ohh, good question. [Looks like yes](https://github.com/CenterForOpenScience/ember-share/blob/develop/app/controllers/sources.js#L23), sorting sources by `long_title`. Come to think of it, `ordering_fields` above should be that instead of `title`, which isn't a field.